Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.6
+
+import PackageDescription
+
+let package = Package(
+    name: "MessageKit",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "MessageKit",
+            targets: ["MessageKit"]
+        )
+    ],
+    dependencies: [
+        .package(url: "https://github.com/nathantannar4/InputBarAccessoryView.git", exact: "4.3.3")
```

Comment:
this is the last version of this dependency that supports iOS 10, but it unfortunately still uses `RxSwift` under the hood, so that dependency is added automatically. When we drop iOS 10 we can bump this version and get rid of `RxSwift` as a result as well

Reply:
logged here https://www.notion.so/monzo/MessageKit-introduces-RxSwift-depedency-d4e3512f13be4977928bea88b0513280

Reply:
@timmikeljmonzo Did this transitive dependency exist already with the CocoaPod?